makerules/modules.mk

```diff
@@ -38,11 +37,10 @@ all: $(MAKE_TARGETS)
 
 # Configure `make` dependencies via their timestamp.
 $(PREBUILD):    $(shell eval $(MKDIR) $(DERIVED_MODULE_DIR))
-$(FORMAT):      $(PREBUILD)
-$(BUILD):       $(FORMAT)
+$(LINT):        $(PREBUILD)
+$(BUILD):       $(PREBUILD)
 $(UNIT):        $(BUILD)
 $(INTEGRATION): $(BUILD)
-$(LINT):        $(BUILD)
 $(IMAGES):      $(BUILD)
 $(SYSTEM):      $(BUILD)
```

Comment:
Retain the `format` target for easy reformatting?
I'm OK with removing the dependencies though.

Reply:
That's what the clojure alias is for - make isn't for general scripting

Reply:
Then maybe restore this comment?

# Run `clojure -M:format` in this directory when this fails.

Even better: have a format check failure describe the remedy.

Reply:
Love markdown.